### ISSUE TITLE
download speed

### ISSUE THREAD

**Issue by 3052** [emarsden/dash-mpd-cli#25]
is an option available to show the current download speed? all I see currently is the elapsed time and a progress bar.

**Comment by emarsden** [comment]
Not currently, but it should be easy to add in the progress bar area.

**Comment by 3052** [comment]
yeah that is must needed. currently I am using FFmpeg or YT-DLP which both have it.

**Comment by emarsden** [comment]
Implemented in v0.2.10.
